css/search.css

```diff
@@ -0,0 +1,96 @@
+   * {
+  box-sizing: border-box;
+}
+.search {
+  width: 100px;
+  height: 100px;
+  margin: 40px auto 0;
+  background-color: #00FEDE;
+  position: relative;
+  overflow: hidden;
+  transition: all 0.5s ease;
+  border-radius: 70px;
+}
+.search:before {
+  content: '';
+  display: block;
+  width: 3px;
+  height: 100%;
+  position: relative;
+  background-color: #00FEDE;
+  transition: all 0.5s ease;
+}
+.search.open {
+  width: 420px;
+  border-radius: 70px;
+}
+.search.open:before {
+  height: 60px;
+  margin: 20px 0 20px 30px;
+  position: absolute;
+}
+
+.search-box {
+  width: 75%;
+  height: 100%;
+  box-shadow: none;
```

Comment:
the box-shadow: none does nothing, the input does not have a box-shadow by default, sorry if i'm mistaken :)